tasks/downloadArchiwalnyBdotTask.py

```diff
@@ -49,7 +47,7 @@ def finished(self, result):
                 QgsMessageLog.logMessage('finished with false')
             else:
                 QgsMessageLog.logMessage("exception")
-                raise self.exception
+                raise ConnectionError(self.exception)
```

Comment:
Może jednak się tego pozbędziemy, bo tak średnio to wyglada jak nagle jest bład python'a 

![image](https://github.com/user-attachments/assets/8569cc0a-3e54-4bb3-9b2d-66aadac35540)